api/__tests__/__snapshots__/project-profile.spec.ts.snap

```diff
@@ -319,7 +319,14 @@ Array [
   Array [
     Object {
       "text": "
-      SELECT DISTINCT(profile.id), profile.* FROM profile
+      SELECT 
+        DISTINCT(profile.id),
```

Comment:
Do you need DISTINCT here? I think `id` is the pk and is guaranteed to be unique.